src/gsi/read_fl_hdob.f90

```diff
@@ -533,6 +533,11 @@ subroutine read_fl_hdob(nread,ndata,nodata,infile,obstype,lunout,gstime,twind,si
                write(6,*) 'READ_FL_HDOB: bad lat/lon values: ', obsloc(1,1),obsloc(2,1)              
                cycle loop_readsb2     
            endif
+!          Convert degree West to degree East for the HDOBs data location for
```

Comment:
Why is this section of code needed?  A spot check of other observation readers did not find the conversion of degrees west to degrees east limited to .not.regional.   Instead, the conversion was applied independent of the value of logical variable regional.

Reply:
Why is the section of code needed?

Reply:
Please see Xingren's response (sent in email)

Reply:
Xingren's email does not get to the root cause.  Is is true that longitude are encoded differently for real-time HWRF and GFS hdob dumps?  If "yes", why?   It seems the encoding of longitude should be consistent for the same data dumped in different networks.   If the answer to the question is "no", then it seems we do not and should not treat longitude differently between regional and .not.regional cases.

Reply:
Forwarded comments to Xingren.

Reply:
Tests with the v16 nam gsi.x indicate that lines 538 and 540 can be removed.  That is, line 539 - the check on negative longitude - may be applied to both regional and global domains.   This result needs to be confirmed with v16 hwrf gsi.x tests.

Reply:
Confirmed with hwrf gsi.x, lines 538 and 540 can be removed.

Reply:
Removed. Committed changes.